site/content/en/docs/Guides/Client SDKs/local.md

```diff
@@ -129,7 +129,7 @@ is to have them both run using the host network, like so:
 In one shell run:
 
 ```shell
-docker run --network=host --rm gcr.io/agones-images/agones-sdk:{{< release-version >}} --local
+docker run --network=host --rm us-docker.pkg.dev/agones-images/release/agones-sdk:{{< release-version >}} --local
```

Comment:
The website changes should be behind a release guard. 

https://agones.dev/site/docs/contribute/#within-a-page